src/Orchard.DisplayManagement/Razor/ThemeAwareViewLocationExpander.cs

```diff
@@ -17,36 +17,47 @@ public class ThemeAwareViewLocationExpander : IViewLocationExpander
         /// <inheritdoc />
         public void PopulateValues(ViewLocationExpanderContext context)
         {
+            var themeManager = context.ActionContext.HttpContext.RequestServices.GetService<IThemeManager>();
+            context.Values["Theme"] = themeManager.GetThemeAsync().Result.Id;
```

Comment:
`themeManager.GetThemeAsync().GetAwaiter().Result.Id` in case ThemeManager throws. See http://stackoverflow.com/a/17284612/235591